### PR TITLE
Updated parameters to read minPredictionConfidenceLevel{1,2}

### PR DESCRIPTION
**Pull request by mayhewsw**
This fixes issue #123 

Also included a couple of comments regarding other params which can be removed.

### CODE REVIEW

File: ner/src/main/java/edu/illinois/cs/cogcomp/ner/LbjTagger/Parameters.java [hunk 1]
```diff
@@ -238,15 +238,29 @@ public static ParametersForLbjCode readAndLoadConfig(ResourceManager rm, boolean
                 param.treatAllFilesInFolderAsOneBigDocument =
                         Boolean.parseBoolean(rm.getString("treatAllFilesInFolderAsOneBigDocument"));
             }
+
+            // TODO: Not ever used. Remove this parameter completely.
```
Comment (christos-c) [inline ner/src/main/java/edu/illinois/cs/cogcomp/ner/LbjTagger/Parameters.java]: Why not remove these now?

File: ner/src/main/java/edu/illinois/cs/cogcomp/ner/LbjTagger/Parameters.java [hunk 2]
```diff
@@ -238,15 +238,29 @@ public static ParametersForLbjCode readAndLoadConfig(ResourceManager rm, boolean
...
                         Boolean.parseBoolean(rm.getString("thresholdPrediction"));
             }
+
+            // TODO: Not ever used. Remove this parameter completely.
             if (rm.containsKey("predictionConfidenceThreshold")) {
                 param.predictionConfidenceThreshold =
                         Double.parseDouble(rm.getString("predictionConfidenceThreshold"));
             }
 
+            if (rm.containsKey("minConfidencePredictionsLevel1")) {
```
Comment (christos-c) [inline ner/src/main/java/edu/illinois/cs/cogcomp/ner/LbjTagger/Parameters.java]: Do these have default values in the `Configurator`?

**Comment by mayhewsw** [comment]
Good point. I removed those. Hopefully confidence should be a little clearer in NER now...

**Comment by christos-c** [comment]
Great! Will merge when tests pass.

**Pull request merged**